README.md

```diff
@@ -1,40 +1,108 @@
-# BTCPay plugin for NopCommerce - accept Bitcoin payments
+# BTCPay Server – Accept Bitcoin payments in nopCommerce
 
+![BTCPay nopCommerce Banner](nopCommerceAcceptBitcoin.png)
 
 ## Plugin Overview
 
-This plugin allows you to easily integrate Bitcoin payments into your NopCommerce website using BTCPay Server — a free, self-hosted and open-source payment gateway solution designed to revolutionize Bitcoin payments. Our seamless integration with NopCommerce ensures a hassle-free connection to your self-hosted BTCPay Server. 
+This plugin allows you to easily start accepting Bitcoin payments in your nopCommerce e-commerce store using BTCPay Server — a free, self-hosted and open-source payment gateway. 
```

Comment:
maybe add "for Bitcoin" at the end of sentence or before "payment gateway"?

Reply:
Fixed https://github.com/btcpayserver/nopcommerce/pull/6/commits/c7c35b94ad68a2aec6309ba68111cd2d90cbe13e